### ISSUE TITLE
The code about generating the  .json file for the test set of EPIC100

### ISSUE THREAD

**Issue by yxgz** [AllenXuuu/DCR#1]
Hello, thanks for your great project! 
I would like to know if the code about generating the` .json` file for the **test set of EPIC100** is available?
Thanks a lot.

**Comment by AllenXuuu** [comment]
Thanks for your interests!
To generate .json files of EPIC, you need to run ```python extract.py --cfg XXX --resume XXX``` first to store the predicted logits of each model. Then we obtain the final result via late fusion. Since the code cleaning issue, fusion script is not ready to release.
But we release the .json file at https://drive.google.com/file/d/13cC2sAd7YxHRd8S5C8v2dSSmhTLm_rK5/view?usp=sharing.
Wish it can help.
